app/domain/borrower.py

```diff
@@ -0,0 +1,9 @@
+class Borrower:
```

Comment:
I honestly don't have a good alternate but I really dislike shared `model` or `domain` packages. They tend to gather a lot of dead code. I don't have an alt but if you have an idea I'm all ears.

In [rsvp-o-tron](https://github.com/michaeldfallen/rsvp-o-tron) (my pet project I used to learn python) I put the model for each blueprint into each blueprint in a `model.py` file. We could try that? So for example this information comes from the deed_api so in `app/service/deed_api/` there would be a model.py that contains `Borrower`, `LandProperty`, `Lender` etc? what do you think?